### ISSUE TITLE
Consider getting a headstart on restricted/banned instances

### ISSUE THREAD

**Issue by april** [hachyderm/community#11]
I know it's never fun, but there are a bunch of instances out there that you'll likely have to ban at some point.

I suppose it's up to you whether you want to do it now or later, but thought if you did want to jump on it now some reference points could be useful:

https://github.com/chaossocial/about/blob/master/blocked_instances.md (chaos.social)
https://infosec.exchange/about/more#unavailable-content
https://octodon.social/about/more#unavailable-content

**Comment by cparedes** [comment]
+1. It does seem like the blocklist in hachyderm/hack is being updated... but it isn't being applied to the actual server. See here for what happens when you don't block things server-wide https://github.com/hachyderm/hack/issues/13

**Comment by cparedes** [comment]
(I used to be and will likely never will again be a member of hachyderm.io because of the lack of a sizable ban list)

**Comment by quintessence** [comment]
👋  Thank you for sending these over! We've actually already incorporated these (looking at what we've deployed). We noticed the blocklist we were maintaining for visibility on the hack repo was falling out of sync with what we were actually deploying on our infrastructure as we are frequently adding domains based on user reports, what we see, and other sources. We're having internal conversations about where we want the public facing list to be hosted in the docs so that people can take a look for their own reference, use on their instances, etc. that will be kept updated.
